src/synapse/api.py

```diff
@@ -223,3 +261,133 @@ def get_version() -> str:
             f"server_version has unexpected content: {server_version}"
         )
     return version_match.group(1)
+
+
+def get_access_token(user: User, server: str, access_token: str) -> str:
+    """Get access token from a user.
+
+    Args:
+        user: user to get access token from.
```

Comment:
Is it from or for?

Reply:
From. This is like the admin user is making a login as another user so it will get an access token from this other user.

Reply:
I changed the parameter name and added a description to the docstring to make it clear what happens.